src/LineChart/store/LineChartActions.ts

```diff
@@ -112,12 +112,9 @@ export const fetchData = (props: LineChartDataHandlerProps) => (dispatch: Dispat
                                 seriesItems[identifier].push(item);
                             }
                             if (seriesEntity.indexOf("/") > -1) {
-                                const seriesGuids = Object.keys(seriesItems);
-                                const constraint = `[id=${seriesGuids.join(" or id=")}]`;
-                                const associatedMxObjects = await fetchByXPath({
-                                    entity: seriesEntity.split("/")[1],
-                                    guid: "",
-                                    constraint,
+                                const guids = Object.keys(seriesItems);
+                                const associatedMxObjects = await fetchByGuids({
```

Comment:
Since we are changing fetchByXPath to fetchByGuids, wouldn't this stop people to work with Xpath constraints ?

Reply:
XPath will always work with a database request, and will not work with NPEs. Using data.get will get the data from cache, if not available from the runtime.